tests/acceptance/suites/vertical-full-page-map.js

```diff
@@ -7,44 +7,44 @@ import Pagination from '../blocks/pagination';
 fixture`Vertical Full Page Map`
   .page(`http://localhost:${PORT}/locations_full_page_map`)
 
-test('Can search and get results', async t => {
-  await SearchBar.submitQuery('virginia');
-  const isResultsPresent = await VerticalResults.isResultsPresent();
-  await t.expect(isResultsPresent).ok();
-});
+// test('Can search and get results', async t => {
```

Comment:
Are these tests all commented out as a result of some local testing changes?

Reply:
I commented out most of the tests so that I could more quickly test the one that is failing on IE11. I will comment them back in when i am done testing